Yubico.Core/src/Yubico/PlatformInterop/Libraries.cs

```diff
@@ -12,12 +12,48 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#if NET47
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+#endif
+
 namespace Yubico.PlatformInterop
 {
     internal static partial class Libraries
     {
 #if NET47
-        internal const string NativeShims = "Yubico.NativeShims.dll";
+        internal const string NativeShims = "Yubico.NativeS hims.dll";
```

Comment:
Extraneous space in `Shims`

Reply:
Apparently I can't resolve my own comment threads? 😂

Reply:
I gotcha